openid-4-verifiable-presentations-1_0.md

```diff
@@ -2325,7 +2325,9 @@ The following is a non-normative example of the payload of a signed OpenID4VP re
 
 ## Response {#dc_api_response}
 
-Every OpenID4VP Authorization Request results in a response being provided through the Digital Credentials API (DC API). The response is an instance of the `DigitalCredential` interface, as defined in [@!W3C.Digital_Credentials_API], and the OpenID4VP Authorization Response parameters as defined for the Response Type are represented as an object within the `data` attribute.
+Every OpenID4VP Authorization Request results in a response being provided through the Digital Credentials API (DC API). The response is an instance of the `DigitalCredential` interface, as defined in [@!W3C.Digital_Credentials_API], and the OpenID4VP Authorization Response parameters as defined for the Response Type are represented as an object within the `data` property.
```

Comment:
```suggestion
Every OpenID4VP Request results in a response being provided through the Digital Credentials API (DC API). The response is an instance of the `DigitalCredential` interface, as defined in [@!W3C.Digital_Credentials_API], and the OpenID4VP Response parameters as defined for the Response Type are represented as an object within the `data` property.
```
we've been avoiding using "authorization request/response" terminology, since authorization request in oauth means a pretty specific grant type..